app/services/doc_auth/response.rb

```diff
@@ -14,7 +14,9 @@ def initialize(
       extra: {},
       pii_from_doc: {},
       attention_with_barcode: false,
-      doc_type_supported: true
+      doc_type_supported: true,
+      # This is a stub to get data paths in place. Replace this as soon as possible.
```

Comment:
Is this comment still relevant? It looks like this is actually implemented pretty close to correctly at this point.

Reply:
Point. Need to add a changelog, too.